lib/srv/usermgmt_linux.go

```diff
@@ -59,21 +59,26 @@ func (*HostUsersProvisioningBackend) LookupGroup(name string) (*user.Group, erro
 	return user.LookupGroup(name)
 }
 
+// LookupGroup host group information lookup by GID
+func (*HostUsersProvisioningBackend) LookupGroupId(gid string) (*user.Group, error) {
```

Comment:
```suggestion
func (*HostUsersProvisioningBackend) LookupGroupID(gid string) (*user.Group, error) {
```
Although technically, you're looking up a group (by ID), not looking up a group ID.